frontend/app/application.py

```diff
@@ -103,12 +103,26 @@ def set_markdown(software, fields):
             software[field] = None
 
 
+def fetch_team_member_data(team_member):
+    person_id = team_member['foreignKey']['id']
+    url = api_url + "/person/%s" % person_id
+    person_dictionary = requests.get(url).json()
+    if "error" in person_dictionary:
+        return team_member
+    team_member.update(person_dictionary)
+    return team_member
```

Comment:
Not entirely sure I understand your intent correctly, but I think this work is already done in https://github.com/research-software-directory/research-software-directory/blob/9d1ef5625366d1a322ad052ea6dd3794745eee90/harvesting/cache.py#L61-L66

Reply:
The goal of the function fetch_team_member_data is to get the name strings of the people in the team of the project. The function cache_project retrieves the data of a project using its key but does not resolve the foreign keys in the team data within the project data structure.

The name strings were necessary for solving issue #452  but they were made unnecessary by issue #575 . So this additional code can also be removed if it causes confusion.